### PR TITLE
Update Audit Rules OVAL

### PR DESCRIPTION
**Pull request by redhatrises**
- Handle optimized audit rules
- Handle -F key=string

### CODE REVIEW

File: shared/oval/audit_rules_auditctl.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="audit_rules_auditctl" version="1">
+    <metadata>
+      <title>Record Any Attempts to Run semanage</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>Audit rules about the information on the use of semanage is enabled.</description>
```
Comment (shawndwells) [inline shared/oval/audit_rules_auditctl.xml]: ```<description>``` needs to be updated

File: shared/oval/audit_rules_augenrules.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="audit_rules_augenrules" version="1">
+    <metadata>
+      <title>Record Any Attempts to Run semanage</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>Audit rules about the information on the use of semanage is enabled.</description>
```
Comment (shawndwells) [inline shared/oval/audit_rules_augenrules.xml]: ```description``` needs to be updated

File: shared/oval/audit_rules_dac_modification_chmod.xml [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
       <!-- Test the augenrules case -->
       <criteria operator="AND">
-        <criterion comment="audit augenrules" test_ref="test_ardm_chmod_augenrules" />
+        <extend_definition comment="audit augenrules" definition_ref="audit_rules_augenrules" />
```
Comment (shawndwells) [inline shared/oval/audit_rules_dac_modification_chmod.xml]: smart

**Comment by redhatrises** [comment]
@shawndwells copy and paste error. It has been fixed. Thanks @shawndwells!

**Comment by shawndwells** [comment]
LGTM. Thanks @redhatrises !

@trevor-vaughan, this should fix your optimized audit rule comments

**Pull request merged**